### PR TITLE
ci: add release script for taos-ws-py

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -77,6 +77,7 @@ jobs:
       #----------------------------------------------
       - name: Install dependencies
         run: |
+          sudo apt update
```
Comment (sangshuduo) [inline .github/workflows/test.yml:80]: Avoid waiting input with "apt update -y" ?

**Review by sangshuduo — APPROVED** [review]
LGTM with a suggestion

File: .github/workflows/test.yml (hunk 1 above)
Comment (zitsen) [reply .github/workflows/test.yml]: Done.

**Review by sunpe — APPROVED** [review]
LGTM

**Pull request merged**